zeppelin-distribution/src/assemble/distribution.xml

```diff
@@ -57,7 +55,6 @@
         <include>README.md</include>
         <include>LICENSE*</include>
         <include>NOTICE</include>
-        <include>DISCLAIMER</include>
```

Comment:
this file is required during the project incubating stage, not required after graduating as TLP(top-level project)